src/docs/stan-reference/distributions.tex

```diff
@@ -118,7 +118,7 @@ \section{Cumulative Distribution Functions}
 \ = \
 \mbox{Pr}[Y < y]
 \ = \
-\int_{-\infty}^y p(y \, | \, \theta) \ \mathrm{d}\theta.
+\int_{-\infty}^y p(y \, | \, \theta) \ \mathrm{d}y.
```

Comment:
This should really be a different bound variable `y'`.  I can just update this in the next round of updates.